svc-task/thandle/thandle.go

```diff
@@ -159,6 +162,19 @@ func (ts *TasksRPC) DeleteTask(ctx context.Context, req *taskproto.GetTaskReques
 		return nil
 	}
 	rsp.Header["Allow"] = "DELETE"
+	if task.PercentComplete == 100 {
+		delErr := ts.deleteCompletedTask(req.TaskID)
+		if delErr != nil {
+			log.Printf("Error while deleting the completed task: %v", delErr)
+			rsp.StatusCode = http.StatusInternalServerError
+			rsp.StatusMessage = response.InternalError
+			rsp.Body = generateResponse(common.GeneralError(http.StatusInternalServerError, response.InternalError, delErr.Error(), nil, nil).Body)
```

Comment:
use fillProtoResponse function for the error if conditions. It will reduce the lines of code and make it more readable

Reply:
done